simulation/Makefile

```diff
@@ -38,6 +38,7 @@ endif
 
 include vcs.mk
 include palladium.mk
+include pldm_diff.mk

```

Comment:
what is pldm_diff.mk?

Reply:
makefile script for difftest on palladium